web/src/redux/apis/Tracetest/Tracetest.api.ts

```diff
@@ -0,0 +1,27 @@
+import {BaseQueryFn, FetchArgs, FetchBaseQueryError, createApi, fetchBaseQuery} from '@reduxjs/toolkit/query/react';
+import {TracetestApiTags} from 'constants/Test.constants';
+import Env from 'utils/Env';
+
+const customBaseQuery: BaseQueryFn<string | FetchArgs, unknown, FetchBaseQueryError> = async (
+  args,
+  api,
+  extraOptions
+) => {
+  const baseUrl = Env.get('baseApiUrl');
+
+  return fetchBaseQuery({
```

Comment:
Thinking ahead a little bit, could we use something like this to inject the tokens or other things coming from the top source?

Reply:
Yes, that is the idea. Use this customBaseQuery to inject tokens, tenantId, etc.